http/static/js/season_qualifiers.js

```diff
@@ -83,45 +91,47 @@ async function do_delete(id) {
     }
 }
 
-function build_rows(qualifiers) {
-    const template = document.querySelector('template#qualifier_submission');
-    var seen = {place: 1, players: {}};
-    let rows = qualifiers.map(e => build_row(template, e, seen));
-    return rows;
-}
-
 document.addEventListener('DOMContentLoaded', async () => {
     let container = document.getElementById('qualifiers');
     let table = document.getElementById('qualifiers_table')
     let season_id = container.dataset['seasonId'];
     let tbody = document.querySelector('#qualifiers_table tbody');
-    let button = document.querySelector('button#toggle-obsolete-button');
-
+    let toggle_obsolete_button = document.querySelector('button#toggle-obsolete-button');
+    let wrapper = document.getElementById('qualifiers-wrapper');
+    let no_qualifiers = document.getElementById('no-qualifiers');
     let qualifiers = await get_qualifiers(season_id);
     try {
         var obsolete_hidden = true;
         function rebuild() {
             tbody.innerHTML = "";
             let rows = build_rows(qualifiers);
+            if (rows.length == 0) {
+                no_qualifiers.classList.remove('hidden');
```

Comment:
Probably should be `add`, here instead

Reply:
And I can't believe you're using snake casing in javascript :pensive:

Reply:
oh yeah i always forget about that